module/actor.js

```diff
@@ -362,6 +362,11 @@ class DCCActor extends Actor {
       }
     ]
 
+    // Initiative: A warrior add his class level to his initiative rolls.
+    if ((token._actor.system.class.className == "Warrior") && (game.settings.get('dcc', 'automateWarriorInititative'))) {
```

Comment:
token._actor.system... could be this.system... as used elsewhere in this function

Reply:
Fixed.